### ISSUE TITLE
Add new work by batch missing data cells in table

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6808]
### Descriptive summary
On dev.nurax.samvera.org (Hyrax 5.0.1), the `Collections` table on the `Relationships` tab is missing data cells under the `Action` header and does not meet WCAG AA requirements

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Make sure that batch upload is enabled
3. Go to https://dev.nurax.samvera.org/batch_uploads/new?locale=en&payload_concern=GenericWork#relationships and run the Site Improve browser extension

![batch_add_no_data_cells](https://github.com/samvera/hyrax/assets/32644254/0cdb9666-941f-40e1-bd5e-2ae7c55ac94b)

**Comment by rjkati** [comment]
This issue is also present on the following pages:

- [Add new work (relationships tab)](https://dev.nurax.samvera.org/concern/generic_works/new?locale=en#relationships) - Action header
- [Edit work (relationships tab)](https://dev.nurax.samvera.org/concern/generic_works/70795791z/edit?locale=en#relationships) - Action header

If it is preferable to create separate tickets for each occurrence, please comment here.

**Comment by christhepianist** [comment]
Tested and confirmed that all three locations are no longer showing this issue.
